content/articles/how-to-add-pure-css-wave-animations-to-your-website/index.md

```diff
@@ -0,0 +1,254 @@
+
+### Introduction
+Adding animations can take you to the next level by making your website seem livelier and more attractive to your users. developers use animations to develop websites that attract more users, increase a websites presence and improve conversions. The downfall of adding animations to a website using JavaScript, as this is the widely used language in adding dynamicity to a website, is animations reduce the performance of a website, especially when the developer fails to select an optimized library. However, there is a turnaround to such a problem, and that is using CSS and HTML only. 
+
+### Goal
+In this article, we will add pure CSS and HTML animations to a website as shown in the video below:
+<iframe width="478" height="269" src="https://www.youtube.com/embed/sAjpeUh3bIs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Insert this video properly! You may check makrdown docs.

Reply:
Hi, it''s impossible to embed YouTube videos .Just give readers a link as I had said to you on Slack!